packages/flutter/lib/src/painting/image_stream.dart

```diff
@@ -911,6 +911,80 @@ class OneFrameImageStreamCompleter extends ImageStreamCompleter {
   }
 }
 
+/// An iterator which iterates over the frames of an image.
```

Comment:
nit: I'd remove "An iterator which" and just start with "Iterates over ..."

Reply:
Done.